projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-cart.ts

```diff
@@ -23,6 +23,23 @@ export function clickOnEditConfigurationLink(cartItemIndex: number): void {
     });
 }
 
+/**
+ * Clicks on the 'Edit Configuration' link in add-to-cart dialog.
+ *
+ */
+export function clickOnEditConfigurationLinkInAddToCartDialog(): void {
+  cy.get('cx-added-to-cart-dialog').within(() => {
+    cy.get('cx-configurator-cart-entry-info').within(() => {
+      cy.get('cx-configure-cart-entry')
+        .find('a:contains("Edit")')
```

Comment:
we should avoid searching elements using translated text. In this case wouldn't it be sufficient to click for any/first a tag inside? There is only one, or?